Installer/package.txt

```diff
@@ -155,7 +155,7 @@ component Misc
 			textfile "ide:License Agreement.txt"
 			textfile "ide:about.txt"
 		if TargetEdition is "CommunityPlus" then
-			textfile "private:License Agreement.txt"
+			textfile "private:communitypluslicense.txt" as "License Agreement.txt"
```

Comment:
Can we call the license file "CommunityPlus License Agreement.txt" in the private repo? That makes it consistent with livecodeforfm :)

Reply:
sure